### PR TITLE
Anti fraud provider API Spectral errors fix

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [X] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to fix spectral errors (Anti-fraud Provider API) - EDU 12755

### CODE REVIEW

File: VTEX - Antifraud Provider API.json [hunk 1]
```diff
@@ -1472,19 +1434,18 @@
...
                     "applicationId": {
                         "type": "string",
-                        "description": "This value is always identified as `vtex`."
+                        "description": "This value is always identified as `vtex`.",
+                        "example": "vtex"
                     },
                     "returnUrl": {
                         "type": "string",
-                        "description": "VTEX website URL. You will redirect the user after they complete login on the payment provider's website. The URL should contain a query string parameter called `authorizationCode` which will be passed empty and which you must fill in before returning the user."
+                        "description": "VTEX website URL. You will redirect the user after they complete login on the payment provider's website. The URL should contain a query string parameter called `authorizationCode` which will be passed empty and which you must fill in before returning the user.",
```
Comment (Isabella-Veloso) [inline VTEX - Antifraud Provider API.json:1446]: Acredito que no final deveria ser "returning **to** the user".

**Review by Isabella-Veloso — COMMENTED** [review]
Tem um comentário.

File: VTEX - Antifraud Provider API.json (hunk 1 above)
Comment (ricardoaerobr) [reply VTEX - Antifraud Provider API.json]: Corrigido.

**Review by Isabella-Veloso — APPROVED** [review]
LGTM

**Pull request merged**